src/inverseRobot.js

```diff
@@ -7,7 +7,16 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const invertedRobot = {};
+
+  for (const key in robot) {
+    if (invertedRobot[robot[key]]) {
```

Comment:
It's good that you are checking for duplicate values, but remember that if `robot[key]` is falsy (like 0, '', or false), this condition will incorrectly return null. A safer way to check if a property exists in an object is to use the `hasOwnProperty` method.